src/commands/create.c

```diff
@@ -22,6 +22,10 @@
 
 extern struct start_data start_data;
 
+/* ignore -pedantic to cast handle_option_console, a function pointer, to a
+ * void* */
+#pragma GCC diagnostic push
```

Comment:
@grahamwhaley - I agree, not generally a fan of pragmas, but aside from changing `-std=`, I don't know how we can suppress these warnings.

Other thoughts on pragmas / options to suppress:

- will static analysis tools understand them?
- will other compilers understand them?